### PR TITLE
DarkheartThicket/ShadeOfXavius: use GetBossTarget() for Growing Paranoia to give a warning sooner

### PR DESCRIPTION
**Pull request by Zireael-N**

### CODE REVIEW

File: Legion/DarkheartThicket/ShadeOfXavius.lua [hunk 1]
```diff
@@ -36,20 +35,21 @@ end
...
-	--self:CDBar(args.spellId, 26) -- pull:25.5, 26.8, 28.0, 19.5 / hc pull:27.1, 32.8, 26.7, 27.9 / m pull:25.5, 37.6, 47.3
-end
-
-function mod:GrowingParanoiaApplied(args)
-	self:PrimaryIcon(args.spellId, args.destName)
-	if self:Me(args.destGUID) then
-		self:Say(args.spellId)
+do
+	local function printTarget(self, player, guid)
+		self:TargetMessage(200289, player, "Urgent", "Alert", nil, nil, true)
```
Comment (elvador) [inline Legion/DarkheartThicket/ShadeOfXavius.lua]: Is the change of color and sound for the Message intended?

File: Legion/DarkheartThicket/ShadeOfXavius.lua [hunk 2]
```diff
@@ -36,20 +35,21 @@ end
...
+		if self:Me(guid) then
+			self:Say(200289)
+		end
+	end
+	function mod:GrowingParanoia(args)
+		self:GetBossTarget(printTarget, 0.4, args.sourceGUID)
+		--self:CDBar(args.spellId, 26) -- pull:25.5, 26.8, 28.0, 19.5 / hc pull:27.1, 32.8, 26.7, 27.9 / m pull:25.5, 37.6, 47.3
+	end
+	function mod:GrowingParanoiaRemoved()
+		self:PrimaryIcon(200289)
```
Comment (funkydude) [inline Legion/DarkheartThicket/ShadeOfXavius.lua]: Can use spellId here.

File: Legion/DarkheartThicket/ShadeOfXavius.lua (hunk 1 above)
Comment (Zireael-N) [reply Legion/DarkheartThicket/ShadeOfXavius.lua]: Nope, my bad.

**Comment by Zireael-N** [comment]
I force-pushed an update to my branch to fix what you've mentioned, not sure if GitHub allows you to squash commits before merging them.

**Comment by elvador** [comment]
We squash every PR before merging, so there is no need to force push ever.
Since we've got a merge conflict now anyway, this will be merged after #145.

For the future (not sure how the other think about it), I'd recommend one PR per boss. Multiple PRs per boss will almost always result in conflicts.

**Pull request merged**